spec/controllers/image_interactives_controller_spec.rb

```diff
@@ -52,11 +52,10 @@
           expect(int.url).to eq(new_values_hash[:url])
         end
 
-        it 'returns to the edit page with a message indicating success' do
+        it 'returns to the edit page' do
           new_values_hash = { :caption => 'I made this up', :url => 'http://mw.concord.org/modeler/_assets/img/mw.png' }
           post :update, :id => int.id, :page_id => page.id, :image_interactive => new_values_hash
           expect(response).to redirect_to(edit_activity_page_path(activity, page))
-          expect(flash[:notice]).to eq('Your image was updated.')
```

Comment:
I'm surprised this changed in this PR. Was this test randomly failing?

Reply:
No, it failed because I removed setting of the `flash[:notice]` in `InteractivesController` and `ImageInteractivesController` extends `InteractivesController`. I removed `flash` related code, as these messages were never displayed anyway and it was creating a false impression of error handling. ;)

Reply:
ok